### ISSUE TITLE
empty GSI when using the @index directive with multiple sortKeys

### ISSUE THREAD

**Issue by gkpty** [aws-amplify/amplify-category-api#496]

**Comment by josefaidt** [comment]
Hey @gkpty :wave: thanks for raising this! This seems related to #490, referencing [this comment](https://github.com/aws-amplify/amplify-category-api/issues/490#issuecomment-1143698907) do the items eventually get populated after a few minutes? Or only after a subsequent mutation?

Taking a further look at this one 🙂

**Comment by josefaidt** [comment]
Marking as a duplicate of #490 and closing in favor of tracking the existing item
